js/base-tool.js

```diff
@@ -549,6 +549,149 @@ function baseTool () {
 				});
 			},
 		},
+		{
+			name: "Mass-Modify Pages",
+			desc: "Quickly modify multiple pages.",
+			html: `
+				<div id="d20plus-mass-page-modify" title="Better20 - Mass-Modify Pages">
+					<div name="modify-pages-list">
+						<div class="list" style="transform: translateZ(0); max-height: 490px; overflow-y: scroll; overflow-x: hidden; margin-bottom: 10px;"><i>Loading...</i></div>
+					</div>
+					<hr>
+                    <p><label class="ib"><input type="checkbox" class="select-all"> Select All</label></p>
+					<table name="modifications-table" style="border-collapse: separate; border-spacing: 10px;">
+					<thead>
+						<tr>
+							<th>Overwrite</th>
+							<th>Label</th>
+							<th>Input</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr>
+							<td><input type="checkbox" name="apply_grid_type"></td>
+							<td><label for="grid_type">Grid Type:</label></td>
+							<td><select name="grid_type">
+								<option selected="" value="square">Square</option>
+								<option value="hex">Hex (V)</option>
+								<option value="hexr">Hex (H)</option>
+								<option value="dimetric">Dimetric</option>
+								<option value="isometric">Isometric</option>
+								</select>
+							</td>
+						</tr>
+						<tr>
+							<td><input type="checkbox" name="apply_diagonaltype"></td>
+							<td><label for="diagonaltype">Diagonal Type:</label></td>
+							<td><select name="diagonaltype">
+								<option class="squareonly" selected="" value="foure" style="">D&amp;D 5E/4E Compatible</option>
+								<option class="squareonly" value="threefive" style="">Pathfinder/3.5E Compatible</option>
+								<option class="squareonly" value="manhattan" style="">Manhattan</option>
+								<option class="hexonly" value="hex" style="display: none;">Hex Path</option>
+								<option value="pythagorean">Euclidean</option>
+								</select>
+							</td>
+						</tr>
+						<tr>
+							<td><input type="checkbox" name="apply_lightrestrictmove"></td>
```

Comment:
Wrap these checkboxes in `<label>`s, such that clicking anywhere in the cell toggles the box

```html
<td><label><input type="checkbox" name="apply_lightrestrictmove"></label></td>
```

etc.

Reply:
would also be nice to center the text; we have a `text-center` class, so:

```html
<td class="text-center"><label><input type="checkbox" name="apply_lightrestrictmove"></label></td>
```

would do it

Reply:
Thanks for the detailed suggestion, done